test/Swashbuckle.AspNetCore.IntegrationTests/SwaggerUIIntegrationTests.cs

```diff
@@ -74,6 +75,28 @@ public async Task SwaggerUIMiddleware_ReturnsInitializerScript(
             Assert.DoesNotContain("%(Interceptors)", jsContent);
         }
 
+        [Theory]
+        [InlineData(typeof(Basic.Startup), "/index.js", null)]
+        [InlineData(typeof(CustomUIConfig.Startup), "/swagger/index.js", "customPlugin1,customPlugin2")]
+        public async Task IndexUrl_DefinesPlugins_IfConfigured(Type startupType, string indexJsPath, string customPlugins)
+        {
+            var client = new TestSite(startupType).BuildClient();
+
+            var jsResponse = await client.GetAsync(indexJsPath);
+            Assert.Equal(HttpStatusCode.OK, jsResponse.StatusCode);
+
+            var jsContent = await jsResponse.Content.ReadAsStringAsync();
+            if (string.IsNullOrEmpty(customPlugins))
+            {
+                Assert.DoesNotContain("\"plugins\"", jsContent);
+            }
+            else
+            {
+                string pluginsNames = string.Join(',', customPlugins.Split(',').Select(p => $"\"{p}\""));
+                Assert.Contains($"\"plugins\":[{pluginsNames}]", jsContent);
+            }
```

Comment:
Rather than have this `if` and having to compute the expected value, split this into two tests and in the second case just specify the literal string expected.